Numbers.java

```diff
@@ -0,0 +1,109 @@
+package Lesson_12;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+
+public class Numbers {
+    ArrayList<Integer> numbersList = new ArrayList<>();
+    char choice;
+    boolean isPrime=true;
+
+    public Numbers (){
```

Comment:
Similar like a comment in CoffeeMachine.java. You need to set your parameters in the constructor so that the outside caller can pass arguments when creating a constructor